io/net/bonding.py.data/bonding.yaml

```diff
@@ -16,6 +16,7 @@ Test: !mux
 bond_interfaces: ""
 peer_ip: ""
 peer_interfaces: ""
+peer_password: ""
```

Comment:
Check in readme file about mention of passwordless ssh setup, and change accordingly.
This is for all tests.